pareto/other_models/CM_module/operational/screening_tool.ipynb

```diff
@@ -0,0 +1,801 @@
+      "\n",
+      "Arc cost:                              17615\n",
+      "Disposal cost:                       1481625\n",
+      "Freshwater cost:                     6290587\n",
+      "Treatment cost:                      1429084\n",
+      "Storage cost:                        1096275\n",
+      "Storage revenue:                     1085312\n",
+      "Lithium revenue:                      462006\n",
+      "Beneficial Reuse Revenue:             369605\n",
+      "Net cost:                            8398263\n",
+      "\n",
+      "---- Treatment at node: N06 ----\n",
+      "\n",
+      "Arc cost:                               3806\n",
+      "Disposal cost:                       1103541\n",
+      "Freshwater cost:                     5302452\n",
+      "Treatment cost:                      1722170\n",
+      "Storage cost:                        1755032\n",
+      "Storage revenue:                     1737482\n",
+      "Lithium revenue:                      513365\n",
+      "Beneficial Reuse Revenue:             410692\n",
+      "Net cost:                            7225463\n",
+      "\n",
+      "---- Treatment at node: N07 ----\n",
+      "\n",
+      "Arc cost:                               4122\n",
+      "Disposal cost:                       1447400\n",
+      "Freshwater cost:                     6173243\n",
+      "Treatment cost:                      1448641\n",
+      "Storage cost:                        1174505\n",
+      "Storage revenue:                     1162760\n",
+      "Lithium revenue:                      462006\n",
+      "Beneficial Reuse Revenue:             369605\n",
+      "Net cost:                            8253540\n",
+      "\n",
+      "---- Treatment at node: N08 ----\n",
+      "\n",
+      "Arc cost:                               3965\n",
+      "Disposal cost:                       1447400\n",
+      "Freshwater cost:                     6173243\n",
+      "Treatment cost:                      1448641\n",
+      "Storage cost:                        1174505\n",
+      "Storage revenue:                     1162760\n",
+      "Lithium revenue:                      462006\n",
+      "Beneficial Reuse Revenue:             369605\n",
+      "Net cost:                            8253383\n",
+      "\n",
+      "---- Treatment at node: N09 ----\n",
+      "\n",
+      "Arc cost:                               3520\n",
+      "Disposal cost:                       1429089\n",
+      "Freshwater cost:                     6110465\n",
+      "Treatment cost:                      1459104\n",
+      "Storage cost:                        1216357\n",
+      "Storage revenue:                     1204193\n",
+      "Lithium revenue:                      462006\n",
+      "Beneficial Reuse Revenue:             369605\n",
+      "Net cost:                            8182731\n",
+      "\n",
+      "Node N06 had the smallest total cost, which amounted to 7636154.943030046.\n",
```

Comment:
```suggestion
      "Node N06 had the smallest total cost, which amounted to 7636154.943030046.\n",
```
This seems incorrect to me - I'm seeing that the net cost for N06 is 7225463, not 7636154.94. What's going on?

Reply:
Thank you for pointing this out. This was just a printing error, I have fixed it - The net cost is indeed 7225463